### PR TITLE
Switch VSCode tasks to use Docker Desktop

### PR DESCRIPTION
**Pull request by sb-msoe**
Adds a .gitattributes file that disables autocrlf because it breaks building in the container.

Changes .vscode/tasks.json to build using Docker Desktop and upload using prosv5.exe.

This eliminates the need for WSL and VMs.

Removes CMakeLists.txt under src folder to fix build error.

### CODE REVIEW

**Review by qprtechman — DISMISSED** [review]
I verified that the script works.

File: .vscode/tasks.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
         {
             "label": "v5_hal Firmware Build",
             "type": "shell",
-            "command": "catkin_make v5_hal_firmware_build",
```
Comment (NathanDuPont) [inline .vscode/tasks.json:12]: If possible I would keep the WSL firmware build as a command, but add it as "v5_hal WSL Firmware Build" for those of us still on the old system

File: .vscode/tasks.json [hunk 2]
```diff
@@ -70,11 +70,11 @@
             }
         },
         {
-            "label": "v5_hal Firmware WSL Upload",
+            "label": "v5_hal Firmware Upload",
             "type": "shell",
-            "command": "catkin_make v5_hal_firmware_wsl_upload",
```
Comment (NathanDuPont) [inline .vscode/tasks.json:75]: Same as above; keeping a WSL upload would be nice if possible

**Review by NathanDuPont — CHANGES_REQUESTED** [review]

File: .vscode/tasks.json (hunk 1 above)
Comment (sb-msoe) [reply .vscode/tasks.json:12]: fixed in latest commit

File: .vscode/tasks.json (hunk 2 above)
Comment (sb-msoe) [reply .vscode/tasks.json:75]: fixed in latest commit

**Review by NathanDuPont — APPROVED** [review]

**Pull request merged**